### ISSUE TITLE
[Feature Request] Plausible deniabiility

### ISSUE THREAD

**Issue by CyberTailor** [tuskyapp/Tusky#2852]
Implement "Hidden accounts" feature. It would work like that:

1. User presses and holds current profile's avatar
2. Activity opens and asks for a PIN code
3. Hidden accounts are now shown in the list

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by mindmeshlink** [comment]
Is there a purpose to this feature? The phrasing of "plausible deniability" makes me wonder what the actual use case is here.

**Comment by CyberTailor** [comment]
Yes, if you're a political dissident in a country with an authoritarian regime (Russia, Belarus, Iran, ...).

**Comment by charlag** [comment]
It might also be useful for some closeted queer people to have hidden accounts but it is a big task, including notifications, share menu etc.

**Comment by realpixelcode** [comment]
> Yes, if you're a political dissident in a country with an authoritarian regime (Russia, Belarus, Iran, ...).

I think it would be too risky for them to store anything like that on a smartphone. They would/should only use Tor browser sessions.

Also, having to enter a PIN doesn't guarantee plausible deniability. It only would if the hidden profile was encrypted in such a way that, even when analysing Tusky's storage, there would be no way of determining whether there's a profile hidden in the encrypted BLOB. That would basically be equivalent to implementing something like VeraCrypt.
